.solhint.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "solhint:recommended",
   "rules": {
-    "compiler-version": ["error", "^0.8.0"],
+    "compiler-version": ["warn", "^0.8.0"],
```

Comment:
@justinkaseman I could not fix this. Please help me getting this back to `error`.